content/roles_and_responsibilities.md

```diff
@@ -1,19 +1,63 @@
 # Roles and Responsibilities
 
 ## Team Organization
+Teams can have up to 5 quizzers on it. Teams can have up to 4 quizzers in the quiz at any given time.
 
 ### Quizzers
 
 ### Captains
+- Scorekeepers must be notified which quizzer is captain and co-captain before the quizzers take their seats to begin the quiz.
+- The team captain and co-captain may change during a given competition but not during a quiz.
+- Only the team captain or co-captain may challenge a decision or respond to a challenge.  If both the captain and co-captain have errored out, another quizzer can be designated by the coach to respond to or initiate challenges.   
 
 ### Coaches
 
 ## Quiz Officials
+- No comment other than “correct” or “incorrect” need be announced by the quiz officials in making a decision.  
+    - However, at the discretion of the answer judge and quizmaster or at the request of the captain, additional information may be given:
+        - If the additional information regarding the ruling is provided at the discretion of the AJ/QM, and not at the request of the captain, this is not considered an explanation of a ruling (which would invalidate a Challenge or Protest).
```

Comment:
This is my addition. Thoughts?    It follows that a QM providing information on a ruling, unprompted, shouldn't invalidate a challenge.

Reply:
Agreed.